drift-codec/src/test/java/com/facebook/drift/codec/URIField.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (C) 2012 ${project.organization.name}
```

Comment:
Header looks wrong

Reply:
Ah. this was generated using the plugin. will fix that.

Reply:
Done